### PR TITLE
About page update

### PR DESCRIPTION
**Pull request by noctera**
| Status  | Type  |
| :---: | :---: |
| :white_check_mark: Ready| Feature |

## Description
In this PR I have refactored the ContributorCard code, in order to add contributors without an Github account to be more flexible.
In addition to this I have added new contributors

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

File: src/Components/ContributorCard/ContributorCard.jsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 import "./ContributorCard.scss";
 
 const Card = ({ name, url, imageUrl, description }) => {
   return (
-    <a
-      href={url ? url : `https://github.com/${name}`}
-      target="_blank"
-      rel="noreferrer noopener"
-    >
+    <a href={url ? url : `/#/about`} target="_blank" rel="noreferrer noopener">
```
Comment (luwol03) [inline src/Components/ContributorCard/ContributorCard.jsx:9]: Use ||

File: src/Components/ContributorCard/ContributorCard.jsx [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
         </div>
-        <img
-          src={imageUrl ? imageUrl : `https://github.com/${name}.png`}
-          alt={name}
-        />
+        {imageUrl ? (
+          <img className="card-image" src={imageUrl} alt={name} />
+        ) : (
+          <div className="card-image-bp">
+            <PersonIcon style={{ transform: "scale(3.5)", color: "#000" }} />
```
Comment (luwol03) [inline src/Components/ContributorCard/ContributorCard.jsx:19]: Use font-size and also use class names

File: src/utils/constants.js [hunk 3]
```diff
@@ -23,25 +23,105 @@ export const rightVocabs = [1, 10];
...
+      url: "https://github.com/noctera",
+      imageUrl: "https://github.com/noctera.png",
+    },
+    {
+      name: "luwol03",
+      description: "16.12.2020",
+      url: "https://github.com/luwol03",
+      imageUrl: "https://github.com/luwol03.png",
+    },
+    { name: "retfloww", description: `08.01.2021\n-\n10.03.2021` },
```
Comment (luwol03) [inline src/utils/constants.js:65]: Does this actually work with the \n? Isn't there some css missing?

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: src/utils/constants.js (hunk 3 above)
Comment (noctera) [reply src/utils/constants.js:65]: This actually works. But yes there is some css missing. I forgot to copy it from the old responsive branch

Comment (luwol03) [reply src/utils/constants.js:65]: I think it's something with pre and wrap.

File: src/utils/constants.js [hunk 4]
```diff
@@ -23,25 +23,105 @@ export const rightVocabs = [1, 10];
 export const numberField = [1, 255];
 
 export const contributors = {
-  desktop: [
-    { name: "noctera", description: "25.11.2020" },
-    { name: "luwol03", description: "17.03.2021" },
-    { name: "zikowang", description: "16.03.2021" },
+  frontend: [
```
Comment (luwol03) [inline src/utils/constants.js:26]: you need to also adjust the translations. These keys are basically the keys for `contributors.xxxx` in the translations.

**Comment by luwol03** [comment]
@noctera can you please resolve conflicts?

**Review by luwol03 — DISMISSED** [review]

**Review by luwol03 — DISMISSED** [review]

**Review by luwol03 — APPROVED** [review]

**Pull request merged**